contrib/svc-cat-apiserver-aggregation-tls-setup.sh

```diff
@@ -59,3 +61,16 @@ echo '{"CN":"'${SVCCAT_SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"r
 export SC_SERVING_CA=${SVCCAT_CA_CERT}
 export SC_SERVING_CERT=apiserver.pem
 export SC_SERVING_KEY=apiserver-key.pem
+
+cat <<EOF >> $HELM_SCRIPT
```

Comment:
one `>` redirect. otherwise helm-install.sh will get bigger and bigger each time.